Sources/Testing/Testing.docc/AddingTags.md

```diff
@@ -10,7 +10,7 @@ See https://swift.org/LICENSE.txt for license information
 See https://swift.org/CONTRIBUTORS.txt for Swift project authors
 -->
 
-Add tags to tests and customize their appearance.
+Use tags to provide semantic information you can access from other tests and test targets.
```

Comment:
I'm not sure what this means? Tags are an organizational tool and we (intend to) allow filtering your `swift test` runs by tag, but I am not sure how that relates to "other tests and test targets".

Reply:
Sure, works for me.